qa/.gitignore

```diff
@@ -1,10 +1,7 @@
-.classpath
-.project
+build
 dbdrivers
 dist
-idl-compiler.*
-build
-TEST-*
-ext/netty-all.jar
+ext/log4j.jar
+ext/netty.jar
 testoutput/
-transaction.log
+TaskImpl.properties
```

Comment:
There are also the following directories:
CR12_0*.log
CosServices.cfg
ObjectStore/
PutObjectStoreDirHere/
 logs/

and files:

 testoutput-*.zip
value_1
value_2